src/libvirt/LibvirtObject.cs

```diff
@@ -70,11 +70,11 @@ private void ThrowExceptionOn(Func<bool> predicate)
             }
         }
 
-        private virError GetLastError()
+        private static virError GetLastError()
```

Comment:
I would not make this static

Reply:
what is your plan about these? check disposed?